package.json

```diff
@@ -19,13 +20,18 @@
   "homepage": "https://github.com/GSA-TTS/identity-dev-docs#readme",
   "dependencies": {
     "@18f/identity-build-sass": "^3.0.0",
-    "@18f/identity-design-system": "^8.1.2"
+    "@18f/identity-design-system": "^8.1.2",
+    "puppeteer": "^22.6.1"
```

Comment:
I think this should be moved to `devDependencies` for consistency with the other dependencies and since it wouldn't be required as part of the standard build of the site.